module/VuFind/src/VuFind/Db/Table/Resource.php

```diff
@@ -265,8 +262,8 @@ public function updateRecordId($oldId, $newId, $source = DEFAULT_SEARCH_BACKEND)
 
             // Deduplicate rows where necessary (this can be safely done outside
             // of the transaction):
-            if (isset($tableObjects['resourcetags'])) {
-                $tableObjects['resourcetags']->deduplicate();
+            if (isset($tableObjects['tagService'])) {
```

Comment:
This needs to be changed back to resourcetags, doesn't it? There's no deduplicate method in the tagService here -- I think we discussed doing this work in a separate PR after merging this one. Am I remembering correctly?

Reply:
That's right, working on it probably will raise a PR by eod.